src/luarocks/cfg.lua

```diff
@@ -145,6 +145,14 @@ end
 cfg.variables = {}
 cfg.rocks_trees = {}
 
+-- some extras for the global enviornment in the config files;
+cfg.os_getenv = os.getenv
+cfg.__dump_env = function()
+  -- debug function, calling it from a config file will show all 
+  -- available globals to that config file
+  print(util.show_table(cfg, "global environment"))
+end
+
```

Comment:
are these necessary? `cfg` is only used internally by LuaRocks where these things are already available.